aoc/day_4.py

```diff
@@ -0,0 +1,39 @@
+import os 
+
+DATA_PATH="inputs/day_4.txt"
+
+def get_assignment_lines():
+    with open(DATA_PATH, 'r') as file:
+        return file.read().splitlines()
+
+def get_sections_list(assignment_row):
+    assignments = assignment_row.split(',')
+    full_assignment = []
+    for assignment in assignments:
+        start = int(assignment.split('-')[0])
+        stop = int(assignment.split('-')[1]) + 1
+        full_assignment.append(list(range(start, stop)))
+    return full_assignment
+
+def is_full_overlap(assignment_pair):
```

Comment:
If you want an exercise with sets, re-write `get_sections_list` to create sets and then update these to use `set` methods.